runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1064,12 +1064,13 @@ TR_SharedCacheRelocationRuntime::validateAOTHeader(TR_FrontEnd *fe, J9VMThread *
       intptr_t featureFlags = generateFeatureFlags(fe);
 
       TR_Version currentVersion;
-      memset(&currentVersion, 0, sizeof(TR_Version));
       currentVersion.structSize = sizeof(TR_Version);
       currentVersion.majorVersion = TR_AOTHeaderMajorVersion;
       currentVersion.minorVersion = TR_AOTHeaderMinorVersion;
-      strncpy(currentVersion.vmBuildVersion, EsBuildVersionString, std::min(strlen(EsBuildVersionString), sizeof(currentVersion.vmBuildVersion) - 1));
-      strncpy(currentVersion.jitBuildVersion, TR_BUILD_NAME, std::min(strlen(TR_BUILD_NAME), sizeof(currentVersion.jitBuildVersion) - 1));
+      strncpy(currentVersion.vmBuildVersion, EsBuildVersionString, sizeof(currentVersion.vmBuildVersion) - 1);
```

Comment:
This is doing unnecessary padding now.

Reply:
The padding done by these uses of `strncpy()` is equivalent to the result via the removed call to `memset()`.

Reply:
Ok, but it could be done better.

Reply:
To clarify, maybe the buffer doesn't need to be entirely cleared, but I'll leave that up to JIT review.